sentinel-dashboard/src/main/java/com/alibaba/csp/sentinel/dashboard/repository/metric/MetricsRepository.java

```diff
@@ -30,14 +30,14 @@
      *
      * @param metric metric data to save
      */
-    void save(T metric);
+    void save(T metric, String app);
 
     /**
      * Save all metrics to the storage repository.
      *
      * @param metrics metrics to save
      */
-    void saveAll(Iterable<T> metrics);
+    void saveAll(Iterable<T> metrics, String app);
```

Comment:
In T, actually `MetricEntity` class, there is a `app` field, it seems that we needn't change the method signature.